compose-builder/get-api-gateway-token.sh

```diff
@@ -24,16 +24,30 @@ if [ "$DEV" = "-dev" ]; then
   CORE_EDGEX_VERSION=0.0.0
 fi
 
-rm -rf keys
-mkdir keys
-openssl ecparam -name prime256v1 -genkey -noout -out keys/ec256.key 2> /dev/null
-cat keys/ec256.key | openssl ec -out keys/ec256.pub 2> /dev/null
+# Init key dir
+GW_KEY_DIR=/tmp/edgex-gateway-keys
```

Comment:
If we're going to make it (key directory) configurable, we might as well write the env var assignment such the the user can override it.

Reply:
Can you explain this a little more? This is a script that's run separate of compose/etc - the user could reconfigure it easily by going in to the script and modifying it?

Reply:
Hi @beaufrusetta, I guess what @bnevis-i meant is that you can define it as env. $GW_KEY_DIR and assign the value you have here as default value.
Syntax with the default value is like `ENV_VAR_NAME=${ENV_VAR_NAME:-/tmp/edgex-gateway-keys}`, in which `/tmp/edgex-gateway-keys` is your default value and if uses wants to override, they can export that env and use it from there.  One example of this can be seen from this: https://github.com/edgexfoundry/edgex-go/blob/master/bin/edgex-docker-launch.sh#L25

Reply:
@beaufrusetta , @bnevis-i , @jim-wang-intel , does this still need to be addressed?

Reply:
yea, i think @bnevis-i 's suggestion is better and more flexible and it should not be too bad to fix, something like: 
```shell
GW_KEY_DIR=${GW_KEY_DIR:-/tmp/edgex-gateway-keys}
```

Reply:
Ok updating...

Reply:
Done.